# sim_parameters.yaml
# Offsets (in degrees) for each joint, to be applied to /teachbot/joint_states_sim before publishing to /teachbot/joint_states

sim_jointstate_publisher:
  ros__parameters:
    
    gui_joint_topic: "/teachbot/joint_states_sim"
    teachbot_joint_topic: "/teachbot/joint_states"

    joint_names:
      - "joint_1_2"
      - "joint_2_3"
      - "joint_3_4"
      - "joint_4_5"
      - "joint_5_6"
      - "joint_6_7"
      - "joint_7_8"
      
    degree_offsets:
      - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.0
 


